src/lib/track.js

```diff
@@ -3,21 +3,36 @@ import { getClientID, getMerchantID } from '@paypal/sdk-client/src';
 
 import { getOrCreateValidCartId, getOrCreateValidUserId } from './local-storage-utils';
 import { getDeviceInfo } from './get-device-info';
+import constants from './constants';
 import type {
     Config,
     TrackingType
 } from './types';
 
+const { imgElementId } = constants;
+
+const updateImgSrc = src => {
+    const oldImg = document.getElementById(imgElementId);
+
+    const newImg = document.createElement('img');
+    newImg.id = imgElementId;
+    newImg.style.display = 'none';
+    newImg.src = src;
```

Comment:
instead of tracking an element `id` (another thing to keep track of) and traversing the dom, why not just create and remove?
```
const newImg = document.createElement('img')
newImg.src....
newImg.onload = function() {
  newImg.remove()
}
newImg.onerror = function() {
  // possibly log fpti event
  newImg.remove()
}
```